urdf2webots/parserURDF.py

```diff
@@ -481,6 +481,9 @@ def getColladaMesh(filename, node, link):
     if hasattr(node, 'material') and node.material:
         for geometry in list(colladaMesh.scene.objects('geometry')):
             for data in list(geometry.primitives()):
+                if str(data.original).split()[0][1:] == 'LineSet':
```

Comment:
```suggestion
                if type(data.original) is lineset.LineSet:
```
This way the test is more self-understandable.
In addition, the collada import should be changed into: `from collada import Collada, lineset`

Reply:
I'll test if that works. One additional note. Yesterday I saw, that Webots actually has lineSets as well. Perhaps instead of excluding them, we turn them into LineSets. Would require a few more lines off code, but should not be too hard.

Reply:
Yes indeed, Webots does support IndexedLineSet: https://cyberbotics.com/doc/reference/indexedlineset
Supporting them completely in the converter, would indeed be the ideal solution. But in the meantime, merging this PR is not bad as it is already much better as it avoids having empty meshes.